### PR TITLE
Update imgair.net,imgtigr.xyz,imglin.xyz

### PR DESCRIPTION
**Pull request by wout-codes**
Link: `http://imgkr.xyz/bm7ja`

### CODE REVIEW

File: EnglishFilter/sections/adservers.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||handtub.com^
```
Comment (zloyden) [inline EnglishFilter/sections/adservers.txt:8]: Missed `$third-party`.

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -435,7 +435,8 @@ upload-4ever.com##a[onclick^="window.open("] ~ a[style*="text-align: left;"][sty
 ||banner-api.grouphfm.com/banner/
 ||zoophiliahentai.com/uploads/banners/
 olx.ro,olx.kz,olx.pl,olx.pt,olx.ua,olx.uz,olx.in,olx.com##div[style="min-height: 120px; display: block;"]
-http$third-party,~image,domain=imageking.xyz|imgkoi.xyz|imgair.net|imgko.xyz|imglin.xyz|imgtigr.xyz
+http$third-party,~image,domain=imageking.xyz|imgkoi.xyz|imgair.net|imgko.xyz|imglin.xyz|imgtigr.xyz|imgkr.xyz
+||imgkr.xyz/shrinker/js/h1j.js
 ||imgtigr.xyz/shrinker/js/h1j.js
 ||imglin.xyz/shrinker/js/h1j.js
 ||imgair.net/shrinker/js/h1j.js
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:442]: You can create a common rule like `/shrinker/js/h1j.js$domain=` with necessary domains.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/adservers.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt:8]: No referer. Please check my commits name

File: EnglishFilter/sections/specific.txt (hunk 2 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:442]: Don't need.

Comment (zloyden) [reply EnglishFilter/sections/specific.txt:442]: Why not? This is more useful with a single rule than creating a bunch of similar ones.

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:442]: -_- Okay fine. Wait up.

File: EnglishFilter/sections/adservers.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/adservers.txt:8]: We are adding this to avoid broken sites in the future. Also this request has a referer,

Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt:8]: Ah yes. Just now it wasn't there. I am not sure.

**Review by zloyden — APPROVED** [review]

**Pull request merged**